doc/en/components/inputs/rating.md

```diff
@@ -22,6 +22,16 @@ npm install {PackageWebComponents}
 
 <!-- end: WebComponents -->
 
+<!-- React -->
+
+First, you need to the install the corresponding {ProductName} npm package by running the following command:
+
+```cmd
+npm install igniteui-react
+```
+
+<!-- end: React -->
+
```

Comment:
The above seems to already exist from line 66. It is currently duplicating:
![image](https://github.com/IgniteUI/igniteui-xplat-docs/assets/15924795/894ef66d-f668-4dd7-ad89-aa57bcced3a3)
Please remove the extra one.